Library/Homebrew/cask/lib/hbc/quarantine.rb

```diff
@@ -12,10 +12,30 @@ def swift
       @swift ||= DevelopmentTools.locate("swift")
     end
 
+    def check_availability_status(command: SystemCommand)
```

Comment:
Use `system_command` directly. We should try to get everything to use `system_command` across the board.

Reply:
👍  While we're at it, may I also update the rest of `quarantine` as well?

Reply:
Sure, go ahead! 😉